plugins/org.komodo.core/src/config/komodo.cnd

```diff
@@ -56,9 +56,6 @@
  * data sources, and other resources created and being worked on by the user.
  */
 [tko:workspace] > nt:unstructured
-  + tko:repositories (tko:repositories) copy
```

Comment:
I think you are right as far as the data sources go. I think `tko:repositories` and `tko:teiids` are a bit different. Those aren't workspace "artifacts" as the others types are. But then again maybe they are. Do you think a Repository instance or a Teiid instance would be something that could be published ad shared from a Komodo Library? I was thinking we would control where those appear in the workspace and the user wouldn't care.